index.js

```diff
@@ -1,7 +1,21 @@
-var assign = require("object-assign");
-var stream = require("stream");
-var babel  = require("babel-core");
-var util   = require("util");
+var assign   = require("object-assign");
+var stream   = require("stream");
+var babel    = require("babel-core");
+var util     = require("util");
+
+var canCompile = require("babel-cli/lib/babel/util").isCompilableExtension;
```

Comment:
This is not a public API, you can't just reach in and grab a random file from a package, this could break at any time. `babel-core` exposes [`.DEFAULT_EXTENSIONS `](https://github.com/babel/babel/blob/777a9ae6e42db1c9d57bbbdca8b233c2dc9b89ac/packages/babel-core/src/index.js#L44) so you should use that.